pkg/controller/backup_controller_test.go

```diff
@@ -340,7 +358,10 @@ func TestProcessBackupCompletions(t *testing.T) {
 					Namespace: velerov1api.DefaultNamespace,
 					Name:      "backup-1",
 					Labels: map[string]string{
-						"velero.io/storage-location": "loc-1",
+						"velero.io/source-cluster-k8s-major-version": "",
+						"velero.io/source-cluster-k8s-minor-version": "",
+						"velero.io/source-cluster-k8s-version":       "v0.0.0-master+$Format:%h$",
```

Comment:
I won't block on this, but it would be nice to replace these magic version strings. It looks like, when you instantiate the `apiServer`, you could do something like:

```
apiServer.DiscoveryClient.FakedServerVersion = &version.Info{
 // initialize the major/minor/git version fields
}
```

And then you could adjust the expected outputs to have the corresponding values.

Reply:
Thanks, Steve. I will add this suggestion when working on the labels during the restore phase.

Reply:
I added the suggestions on the last commit.